advanced/01-pihole.conf

```diff
@@ -41,3 +41,8 @@ log-facility=/var/log/pihole.log
 local-ttl=2
 
 log-async
+
+# Signal to Firefox that the local network is unsuitable for DNS-over-HTTPS
+# This follows https://support.mozilla.org/en-US/kb/configuring-networks-disable-dns-over-https
+# (sourced 7th September 2019)
+server=/use-application-dns.net/
```

Comment:
This will be deleted whenever the configuration is re-processed: https://github.com/pi-hole/pi-hole/blob/1d43c0a2fa36ddc993db51b1751e26cbc8123fbf/advanced/Scripts/webpage.sh#L141